testing/py-google-apputils/APKBUILD

```diff
@@ -22,8 +22,9 @@ build() {
 package() {
 	cd "$builddir"
 	python setup.py install --prefix=/usr --root="$pkgdir" || return 1
+	chmod o+r "$pkgdir"/usr/lib/python2.7/site-packages/google_apputils-"$pkgver"-py2.7.egg-info/* || return 1
```

Comment:
It's a workaround this issue: https://github.com/google/google-apputils/issues/4
Without it, the package is broken, see for instance: https://travis-ci.org/alpinelinux/aports/builds/134998764

I can do that in a seperate commit if you prefer.